code-studio/src/js/components/share_dialog_body.jsx

```diff
@@ -92,6 +92,16 @@ var ShareDialogBody = React.createClass({
         appType={this.props.appType}/>;
     }
 
+    var exportButton = null;
+    if (this.props.appType === 'applab') {
+      // TODO: Can I not proliferate the use of global references to Applab somehow?
```

Comment:
True.  I find pushing globals 'up and out' is often the first step toward making them non-global though, so this might be a good first step.